(AAA) Access for Enterprises guide (#36600)
DataDog/documentation: content/en/getting_started/access_for_enterprises/_index.md

---
title: Access for Enterprises
description: Design and implement an access control strategy for your Datadog deployment at scale.
further_reading:
- link: "/administrators_guide/getting_started/"
  tag: "Documentation"
  text: "Administrator's Guide: Getting Started"
- link: "/getting_started/teams/"
  tag: "Documentation"
  text: "Getting Started with Teams"
- link: "/account_management/rbac/"
  tag: "Documentation"
  text: "Access Control (RBAC)"
---

## Overview

> This guide helps Datadog administrators at large organizations design and implement an access control strategy. It is intended for organizations that fit any of the below profiles:

Reviewer: Removing directional language for accessibility.

```suggestion
This guide helps Datadog administrators at large organizations design and implement an access control strategy. It is intended for organizations that fit any of the following profiles:
```

- Have **100 or more users** across multiple teams or business units
- Operate in **regulated industries** with data sensitivity requirements
- Need to **expand Datadog usage** across more users and teams while maintaining governance

> As organizations grow on Datadog, they face a core tension: enabling self-service for developers, operators, and business users while maintaining boundaries on who can see or change what. This guide provides a decision framework for resolving that tension, not step-by-step UI instructions.

Reviewer: Suggest adjusting the voice to address the user directly (You vs They). I think it reads better to introduce the problem and then follow up with a directive (Use this if...)

```suggestion
As your organization scales on Datadog, it faces the challenges of enabling self-service for developers, operators, and business users while maintaining boundaries on who can view or modify. This following guides provide a decision framework for Datadog administrators at large organizations to design and implement an access control strategy.

Use these guides if your organization:

- Has **100 or more users** across multiple teams or business units
- Operates in a **regulated industry** with data sensitivity requirements
- Needs to **expand Datadog usage** across more users and teams while maintaining governance**
```

> 

Reviewer: ```suggestion
## Prerequisites

This guide assumes familiarity with Datadog basics. Before starting, review:

- [Administrator's Guide: Getting Started][10] for overall platform setup and onboarding
- [Getting Started with Teams][11] for team structure, sync strategies, and ownership workflows
- [Access Control (RBAC)][12] for Datadog's role and permission model
```

> ### What this guide covers

Reviewer: ```suggestion
## Design and implement an access control strategy
```

> Each section addresses a layer of your access control strategy, from organizational structure through enforcement and auditing:

Reviewer: ```suggestion
You can read this guide sequentially to plan a new access strategy, or jump to the section most relevant to your current challenge. 

Each section addresses a layer of your access control strategy, from organizational structure through enforcement and auditing:
```

| Section | What to decide |
| :---- | :---- |
| [Organizing Your Datadog Deployment][1] | Single org vs. multi-org, and when each is appropriate |
| [Permissions and Feature Access][2] | Custom roles, keeping roles current, role hygiene |
| [Assigning Users to Roles and Teams][3] | Roles vs. Teams, and how to assign users at scale |
| [Protecting Assets][4] | Restricting who can edit or view Dashboards, Monitors, and other assets |
| [Protecting Sensitive Data][5] | Restricting access to telemetry data using Data Access Control |
| [Credential Management][6] | Managing API keys, app keys, and short-lived tokens at scale |
| [Creating Access Policies][7] | A reference for choosing the right enforcement mechanism |
| [Sharing Across Organizations][8] | Cross-Org Visibility, Shared Dashboards, and multi-org governance |
| [Example Implementations][9] | Enterprise implementation templates |

### How to use this guide

> You can read this guide sequentially to plan a new access strategy, or jump to the section most relevant to your current challenge. Each section ends with a **Recommendations** block summarizing the key takeaways and a **Further Reading** list linking to detailed documentation.

Reviewer: ```suggestion
```

### Prerequisites

This guide assumes familiarity with Datadog basics. Before starting, review:

- [Administrator's Guide: Getting Started][10] for overall platform setup and onboarding
- [Getting Started with Teams][11] for team structure, sync strategies, and ownership workflows

> - [Access Control (RBAC)][12] for Datadog's role and permission model

Reviewer: Recommend moving this section up to the top of the page.

```suggestion
```

## Next steps

> Start with [Organizing Your Datadog Deployment][1] to determine whether your organization should use a single Datadog org or multiple orgs. This foundational decision shapes every access control decision that follows.

Reviewer: Recommend consistent use of org, multi-org, multiple orgs, multiple organizations.

Reviewer: This is probably belongs in the overview of the actual page. 

```suggestion
Start with [Organizing Your Datadog Deployment][1] to determine whether your organization should use a single Datadog org or multiple orgs.
```
